runtime/plugin/man.vim

```diff
@@ -5,6 +5,12 @@ if exists('g:loaded_man')
 endif
 let g:loaded_man = 1
 
-command! -complete=customlist,man#complete -nargs=* Man call man#open_page_command(<f-args>)
+command! -count=0 -complete=customlist,man#complete -nargs=* Man call man#open_page_command(v:count, v:count1, <f-args>)
 
-nnoremap <silent> <Plug>(Man) :<C-U>call man#open_page_mapping(v:count, v:count1, expand('<cWORD>'))<CR>
+autocmd BufEnter man://*
+      \ if exists('b:manwidth') && line('$') == 1 && empty(getline(1)) |
+      \   unlet b:manwidth |
+      \   silent execute 'Man' matchstr(expand('<amatch>'), 'man://\zs.*') |
+      \ endif
```

Comment:
@justinmk It was a suggestion I made in #5169.  Out of habit, I use `:e` to reload a buffer.  For me, it made sense to do that when I resized the window and wanted the man page to be re-rendered.